src/test/request.test.js

```diff
@@ -74,4 +80,62 @@ describe('Request route endpoints', () => {
       expect(response.body.status).to.equal('success');
     });
   });
+
+  describe('Trip Request Endpoint', () => {
+    it('should successfully create a one-way trip request', async () => {
+      const response = await chai
+        .request(server).post('/api/trip/request').set('Cookie', `token=${adminToken};`)
+        .send(tripRequest);
+      expect(response).to.have.status(201);
+      expect(response.body.data).to.be.a('object');
```

Comment:
We should use `include`
```suggestion
      expect(response.body.data).to.include({//some key/value pairs of the expected object});
```
Do this everywhere this method is used

Reply:
Hey, @Segun-Ogundipe , I need a few clarificiations on your comment. Is the use of `include` call in place of `to.be.a()` a hard rule? I don't see the problem with checking if the data is an object, and then also checking for key pairs included in the object. `to.be.a()` shouldn't be changed outright, except theres a reason not to use that call.

Reply:
This is to enable us check the contents of the object to ensure we are getting what we expect. {} and {key: 'value'} are both objects and will pass the test you have written even though the first object does not contain what we are expecting.

Reply:
ok thank you @amarachukwu-agbo